actions.js

```diff
@@ -51,21 +64,188 @@ module.exports = function (self) {
 				self.sendCam(url, 'POST')
 			},
 		},
+		stop_robot: {
+			name: 'Stop Robot',
+			options: [ ROBOT_OPTION ],
+			callback: async (event) => {
+				const idx = await self.getRobotIndex(event.options.robot)
+				const url = `/v1/robots/${idx}/stop`
+				self.sendCam(url, 'PUT')
+			}
+		},
 		stop_all: {
-			name: 'Stop All',
+			name: 'Stop All Robots',
 			callback: async (event) => {
 				self.sendCam('/v1/robots/stop')
 			},
 		},
 		set_move_option: {
-			name: 'Set Move Option',
+			name: 'Set Move Option (Deprecated)',
```

Comment:
It seems that set_move_option is one case of set_move_option_v2. So no need to deprecate it. Better solution would be to upgrade it. You can simply add the missing options with a upgrade script.